src/main/java/techcourse/myblog/web/AccountController.java

```diff
@@ -0,0 +1,124 @@
+package techcourse.myblog.web;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.*;
+import techcourse.myblog.domain.User;
+import techcourse.myblog.domain.UserForm;
+import techcourse.myblog.domain.UserRepository;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+import java.util.List;
+import java.util.Optional;
+
+@Slf4j
+@Controller
+public class AccountController {
+    private UserRepository userRepository;
+
+    @Autowired
```

Comment:
단일 생성자인 경우, `@Autowired`가 없어도 됩니다.

> If a class only declares a single constructor to begin with, it will always be used, even if not annotated.

```suggestion

```

Reply:
어노테이션 제거했습니다. 확인 부탁드립니다!